# Moa template "mummer"

Copyright 2012, Mark Fiers

This software is under the gplv3 license, please see "COPYING"
for details




## Description

Run mummer between two sequences


## Filesets

input
:    Set 1 input fasta files

     - *type:* set<br>
     - *category:* input<br>
     - *optional:* no

reference
:    Set 1 input fasta files

     - *type:* set<br>
     - *category:* input<br>
     - *optional:* no



## Parameters

organism
:    Organism name - used in the AGP file

     - *type:*String<br>
     - optional:yes
linker
:    linker sequence for the merged output sequence

     - *type:*String<br>
     - optional:yes
taxid
:    Taxonomy id - used in the AGP file

     - *type:*String<br>
     - optional:yes
matchmode
:    use all matching fragments (max) or only unique matchers (mum)

     - *type:*set<br>
     - optional:yes
breaklen
:    Set the distance an alignment extension will attempt to extend poor scoring regions before giving up (default 200)

     - *type:*integer<br>
     - optional:yes
gff_source
:    GFF source field

     - *type:*String<br>
     - optional:yes
genomecenter
:    genome center - used in the AGP file

     - *type:*String<br>
     - optional:yes
base
:    base name for all generated files

     - *type:*String<br>
     - optional:yes
mum_plot_raw
:    plot an alternative visualization where mummer does not attempt to put the sequences in the correct order

     - *type:*boolean<br>
     - optional:yes


## Private parameters

organism
:    Organism name - used in the AGP file

     - *type:*String<br>
     - optional:yes
linker
:    linker sequence for the merged output sequence

     - *type:*String<br>
     - optional:yes
taxid
:    Taxonomy id - used in the AGP file

     - *type:*String<br>
     - optional:yes
matchmode
:    use all matching fragments (max) or only unique matchers (mum)

     - *type:*set<br>
     - optional:yes
breaklen
:    Set the distance an alignment extension will attempt to extend poor scoring regions before giving up (default 200)

     - *type:*integer<br>
     - optional:yes
gff_source
:    GFF source field

     - *type:*String<br>
     - optional:yes
genomecenter
:    genome center - used in the AGP file

     - *type:*String<br>
     - optional:yes
base
:    base name for all generated files

     - *type:*String<br>
     - optional:yes
mum_plot_raw
:    plot an alternative visualization where mummer does not attempt to put the sequences in the correct order

     - *type:*boolean<br>
     - optional:yes


## General information

* Author: Mark Fiers
* Backend: ruff


*This file is automatically generated by "moa_template_setup", part of
 the moa software (see: http://mfiers.github.com/Moa/)
